### PR TITLE
pagination added

### PR DESCRIPTION
**Pull request by azzilian**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -70,6 +70,16 @@
 			<groupId>org.hibernate.validator</groupId>
 			<artifactId>hibernate-validator</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>3.0.0</version>
+		</dependency>
```
Comment (Nikname2303) [inline pom.xml]: Why required that dependency?

File: src/main/java/com/onlinebookstore/onlinebookstore/controller/BookController.java [hunk 2]
```diff
@@ -25,28 +29,34 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Find All Books", description = "Find All Books in DB")
```
Comment (Nikname2303) [inline src/main/java/com/onlinebookstore/onlinebookstore/controller/BookController.java:32]: Maybe should say it list, for potential customer

**Review by Nikname2303 — APPROVED** [review]
Good job, some minor comments

File: pom.xml (hunk 1 above)
Comment (andrii-hoienko) [reply pom.xml]: agree

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Nice, left minor comments

File: pom.xml (hunk 1 above)
Comment (azzilian) [reply pom.xml]: I forgot to remove it after fixed some bugs

File: src/main/java/com/onlinebookstore/onlinebookstore/controller/BookController.java (hunk 2 above)
Comment (azzilian) [reply src/main/java/com/onlinebookstore/onlinebookstore/controller/BookController.java:32]: You mean like in description? I will think about more detailed information maybe will go with other patch

**Review by andrii-hoienko — APPROVED** [review]
gj!

**Pull request merged**